frontend/src/App.js

```diff
@@ -25,8 +29,9 @@ class App extends Component {
                 <div key={history.location.pathname}>
                   <Switch location={history.location}>
                     <Route exact path="/figures/dashboard" component={HeaderContentMaus} />
-                    <Route exact path="/figures/test" component={HeaderContentReportsList} />
-                    <Route exact path="/figures/course" component={HeaderContentCourse} />
+                    <Route exact path="/figures/reports" component={HeaderContentReportsList} />
```

Comment:
Seeing the same code added to two places, do you think it makes sense to make this a component/partial starting with the `<Route render={ ({location})...` ? or does routing not work well in partials?